NuRadioMC/utilities/medium_base.py

```diff
@@ -420,6 +420,211 @@ def get_birefringence_index_of_refraction(self, position):
 
         return nx, ny, nz
 
+class IceModelExponentialPolynomial(IceModel):
+    """
+    predefined ice model (to inherit from) with polynomial exponential shape of ny degree
+    """
+    def __init__(self, a, z_0, z_shift=0*units.meter, z_air_boundary=0*units.meter, z_bottom=None, 
+                 density_factor=0.8506 * (units.cm**3/units.gram)):
+        """
+        initiation of an exponential polynomial ice model.
+
+        The bottom defined here is a boundary condition used in simulations and
+        should always be defined. Note: it is not the same as reflective bottom.
+        The latter can be added using the `add_reflective_layer` function.
+
+        The z_shift is a variable introduced to be able to shift the model
+        up or down along the z direction when this is needed to account for
+        differences along station for example.
+
+        Parameters
+        ----------
+        a:  (n,) np.array of floats, NuRadio density units
+            coefficients for the exponential polynomial describing the density profile
```

Comment:
Should "n" be a fixed number of parameters? I think a bit of math would help here ...